django/test/client.py

```diff
@@ -631,8 +632,15 @@ def login(self, **credentials):
             return False
 
     def force_login(self, user, backend=None):
+        def get_backend():
+            from django.contrib.auth import load_backend
+            for backend_path in settings.AUTHENTICATION_BACKENDS:
+                backend = load_backend(backend_path)
+                if hasattr(backend, 'get_user') and callable(backend.get_user):
```

Comment:
Did you have a use case for `backend.get_user` not being callable? Do any tests fail if that's removed?